### PR TITLE
Add service type std_srvs/Trigger

### PR DESCRIPTION
**Pull request by PetterVMC**

### CODE REVIEW

File: Source/ROSIntegration/Private/Conversion/Services/std_srvs/StdSrvsTriggerRequestConverter.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+	_ServiceType = "std_srvs/Trigger";
+}
+
+bool UStdSrvsTriggerRequestConverter::ConvertOutgoingRequest(TSharedPtr<FROSBaseServiceRequest> Request, bson_t** BSONRequest)
+{
+	auto TriggerRequest = StaticCastSharedPtr<std_srvs::FTriggerRequest>(Request);
+	*BSONRequest = BCON_NEW();
+
+	UE_LOG(LogTemp, Verbose, TEXT("This is rospy tut req converter"));
```
Comment (Sanic) [inline Source/ROSIntegration/Private/Conversion/Services/std_srvs/StdSrvsTriggerRequestConverter.cpp:15]: Could you please delete the UE_LOG line?

File: Source/ROSIntegration/Private/Conversion/Services/std_srvs/StdSrvsTriggerRequestConverter.cpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	return true;
+}
+
+bool UStdSrvsTriggerRequestConverter::ConvertIncomingRequest(ROSBridgeCallServiceMsg &req, TSharedPtr<FROSBaseServiceRequest> Request)
+{
+	//*Request = MakeShareable(new std_srvs::FTriggerRequest);
+	auto ServiceRequest = StaticCastSharedPtr<std_srvs::FTriggerRequest>(Request);
+	
+	UE_LOG(LogTemp, Verbose, TEXT("Received Trigger Request"));
```
Comment (Sanic) [inline Source/ROSIntegration/Private/Conversion/Services/std_srvs/StdSrvsTriggerRequestConverter.cpp:24]: Could you please delete the UE_LOG line?

**Review by Sanic — COMMENTED** [review]
Thanks a lot for the PR!  👍  I have just two minor requests for a change.

**Comment by PetterVMC** [comment]
@Sanic Sure, forgot about those! Just updated the PR with a new commit. 

This service was basically done by copying RospyTutorialsAddTwoInts and modifying necessary parts, which is why those log lines and the commented out line was still there.

**Pull request merged**